src/views/home/home.spec.jsx

```diff
@@ -6,15 +6,18 @@ import Home from './home';
 
 configure({ adapter: new Adapter() });
 
-describe('Hello View', () => {
+describe('Home component', () => {
   const home = shallow(<Home />);
 
   it('should not be null', () => {
     expect(home).not.toBeNull();
   });
 
-  it('should have a container be null', () => {
+  it('should have a container not be null', () => {
     expect(home.find('.home').length).toEqual(1);
   });
 
+  // it('should have a hero-banner Component', () => {
```

Comment:
this should be tested now right, since the Hero Banner is a part of the home page?